packages/client-twitter/src/environment.ts

```diff
@@ -1,6 +1,8 @@
 import { IAgentRuntime } from "@ai16z/eliza";
 import { z } from "zod";
 
+const DEFAULT_MAX_TWEET_LENGTH = 200;
```

Comment:
Why change it to 200 when it's 280 in the original code?

Reply:
ahh you're right it's updated